pkg/converter/constant.go

```diff
@@ -20,6 +20,8 @@ const (
 	LayerAnnotationNydusBootstrap     = "containerd.io/snapshot/nydus-bootstrap"
 	LayerAnnotationNydusSourceChainID = "containerd.io/snapshot/nydus-source-chainid"
 	LayerAnnotationNydusEncryptedBlob = "containerd.io/snapshot/nydus-encrypted-blob"
+	LayerAnnotationSourceDigest       = "containerd.io/snapshot/nydus-source-digest"
```

Comment:
`LayerAnnotationNydusSourceDigest`?

Reply:
If the content store of the acceld provides remote cache, the cache layer in remote cache will have a `LayerAnnotationNydusSourceDigest` annotation to identify this is a converted layer, 
`LayerAnnotationNydusTargetDigest` will identify what is the converted digest, so we can skip this unnecessary conversion.

Reply:
Got it.
How about changing `LayerAnnotationSourceDigest` to `LayerAnnotationNydusTargetDigest`.

Reply:
fixed